gems/auto_select2/RUBYSEC-2016-0001.yml

```diff
@@ -0,0 +1,13 @@
+---
+gem: auto_select2
+rubysec: 2016-0001
+url: https://github.com/Loriowar/auto_select2/issues/4
+title: |
+  Search functionality allows arbitrary search execution
```

Comment:
I personally include the gem name in the title, but not required... We should probably standardize.